### PR TITLE
PYT-1860 Add testing for pyramid 2.0

### PR DESCRIPTION
**Pull request by angelobretenovv**
Pyramid has a new version (2.0) . We need to test that to claim support for it.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,11 @@
 uwsgi_max_extras = {"uwsgi==2.0.*"}
 uwsgi_min_extras = {"uwsgi==2.0.14"}
 gunicorn_min_extras = {"gunicorn==0.16.1"}
-pyramid_extras = {"pyramid<2", "waitress<2.1"} | trigger_extras
+pyramid_extras = {
+    "pyramid<2; python_version < '3.6'",
+    "pyramid==2.0; python_version >= '3.6'",
```
Comment (lazorchakp) [inline setup.py:30]: can you make this `pyramid<3`? that's generally been our strategy for framework versioning

Comment (angelobretenovv) [reply setup.py:30]: Yes I'll change that, I made it that way simply because Pyramid 2.0 doesn't support python 2 anymore

Comment (lazorchakp) [reply setup.py:30]: oh, definitely keep it for python < 3.6, but for the >= 3.6 case you can use pyramid<3

Comment (angelobretenovv) [reply setup.py]: Okay thank you

**Review by lazorchakp — APPROVED** [review]

**Review by danilito19 — APPROVED** [review]

**Pull request merged**